test/functional/test_ivn.py

```diff
@@ -124,21 +138,18 @@ def start_node_with_config(self, node_name, ns_name, boot_image, iso):
                                                      "mac": "00:60:16:93:b9:1a"})
             fake_node["compute"]["networks"][0]["port_forward"] = [{"outside": 8022, "inside": 22, "protocal": "tcp"}]
             fake_node["compute"]["networks"][0]["mac"] = "00:60:16:93:b9:2d"
-        print "fake_node"
-        print fake_node
         fake_node_up = self._start_node(fake_node)
         return fake_node_up
 
     def client_ssh(self, ns_ip):
         ssh = sshclient.SSH(host=ns_ip, username="ubuntu", password="password")
         ssh.wait_for_host_up(300)
-        status, output = ssh.exec_command("ifconfig")
         return ssh
 
     def ping_peer(self, ssh, peer_ip):
         status, output = ssh.exec_command("ping -I eth0 %s  -c 5" % peer_ip)
         print output
-        self.assertIn("0% packet loss", output, "node connection passed!")
+        self.assertIn("0% packet loss", output, "node connection failed!")
```

Comment:
maybe more safe if change to 
self.assertNotIn("100% packet loss", output, "node connection failed!")

since the first ICMP package may be failed in some environment.

Reply:
good point！